specification/columns/columns.mdpp

```diff
@@ -19,6 +19,7 @@ The FOCUS specification defines a group of columns that provide qualitative valu
 !INCLUDE "commitmentdiscountid.md",1
 !INCLUDE "commitmentdiscountname.md",1
 !INCLUDE "commitmentdiscounttype.md",1
+!INCLUDE "commitmentdiscountusage.md",1
```

Comment:
Suggested Column Name: **Commitment Status.**

Value Examples:

- Commitment Status > Used, Not Used
- Commitment Status > Used, Unused

Reply:
We said we want to keep "Discount" in the name to make it specific to commitment discounts, so the only tweak I would suggest is "Commitment Discount Status". I do prefer this tho.